lib/tasks/migration.rake

```diff
@@ -854,6 +871,49 @@ namespace :migration do
 
   end
 
+  def migrate_google_stats(file_dir)
+
+    MigrationLogger.info " +++++++ START: migrate google stats #{file_dir} +++++++ "
+    allfiles = Dir.glob(file_dir+"*.txt")
+    filecount = allfiles.select { |file| File.file?(file) }.count
+    MigrationLogger.info "Files to process: " + filecount.to_s
+    allfiles.sort.each_with_index do |file, thisfile|
+      start_time = Time.now
+      MigrationLogger.info "Processing the file #{file} (#{thisfile + 1} of #{filecount})"
+      index = file.rindex('/')
+      id = file[index+1..-1]
+      id.slice! ".txt"
+     
+      begin 
+        byebug
```

Comment:
This was merged into master without removing. I'm going to remove it in a separate PR.